### DISCUSSION TITLE
Is CURLSSLOPT_NATIVE_CA supported on libressl?

### DISCUSSION THREAD

**Discussion by genotrance** [curl/curl#14869] [Q&A] [answered]
I'm using the Windows [build](https://curl.se/windows/) of libcurl and wanted to use the `CURLSSLOPT_NATIVE_CA ` option. However, the [documentation](https://curl.se/libcurl/c/CURLOPT_SSL_OPTIONS.html) does not explicitly mention libressl. I understand libressl is a fork of openssl but it's been 10 years. Thanks in advance!

**Comment by bagder** [comment, marked as answer]
Yes it is. When the man page says it is for "OpenSSL", that includes all the big known forks: BoringSSL, LibreSSL, AWS-LC and quictls.
